.circleci/config.yml

```diff
@@ -192,6 +204,26 @@ jobs:
           command: xcodebuild -project MapboxDirections.xcodeproj -scheme 'MapboxDirections watchOS' -destination 'platform=watchOS Simulator,name=Apple Watch Series 5 - 44mm,OS=<< parameters.watchOS >>' clean build
       - save-cache
 
+  publish-documentation:
+    macos:
+      xcode: "12.5.1"
+    steps:
+      - checkout
+      - install-mapbox-token
+      - install-carthage
+      - install-mbx-ci
+      - install-gh
+      - run:
+          name: "Publish Documentation"
+          command: |
+            ./scripts/publish-documentation.sh << pipeline.git.tag >>
+            git remote set-url origin "https://x-access-token:$(mbx-ci github writer public token)@github.com/mapbox/mapbox-directions-swift.git"
+            git config user.email "release-bot@mapbox.com"
+            git config user.name "Mapbox Releases"
+            git push --set-upstream origin docs-<< pipeline.git.tag >>
+            export GITHUB_TOKEN="$(mbx-ci github writer public token)"
+            gh pr create --reviewer "mapbox/navigation-ios" --fill
```

Comment:
By the way, do we need a PR? Why dot pushing directly to `publisher-production` as we do in Navigation SDK repo?